stl/inc/yvals_core.h

```diff
@@ -613,6 +613,13 @@
 #endif // ^^^ MSVC ^^^
 #endif // _STL_RESTORE_DEPRECATED_WARNING
 
+#if defined(__clang__) || defined(__CUDACC__) || defined(__INTEL_COMPILER) || defined(__EDG__)
+#define _STL_DECLSPEC_NOINITALL
+#else // ^^^ not MSVC/ MSVC vvv
+#define _STL_DECLSPEC_NOINITALL \
+    __pragma(warning(push)) __pragma(warning(disable : 4845)) __declspec(no_init_all) __pragma(warning(pop))
+#endif // ^^^ MSVC ^^^
+

```

Comment:
We only need this in one header, and temporarily at that. Let's move this to `<format>` and `#undef` it when we're done.